python.d/spigotmc.chart.py

```diff
@@ -52,7 +52,9 @@ def check(self):
             return False
         try:
             self.connect()
-        except (mcrcon.MCRconException, socket.error):
+        except (mcrcon.MCRconException, socket.error) as err:
+            self.error('Error connecting.')
+            self.error(repr(err))
```

Comment:
Why not both in one line?

Reply:
Habit mostly.  `repr(err)` may be multi-line itself, and I generally prefer to separate log messages that may be multi-line so that the first line isn't on the same line as another log message.  I can update it to just do it all with a single call instead if desired.

Reply:
usually less calls -> better :)

I just wonder how we handle multi-line msgs...